.rubocop.yml

```diff
@@ -69,6 +69,7 @@ Metrics/ClassLength:
     - 'app/services/manifest_builder/manifest_service_locator.rb'
     - 'app/services/pdf_generator/cover_page_generator.rb'
     - 'app/decorators/valkyrie/resource_decorator.rb'
+    - 'app/controllers/ephemera_folders_controller.rb'
```

Comment:
For next week: maybe we should increase the class length? Or look at these classes and see if there's any refactoring we want to do?